metrics/metrics.go

```diff
@@ -75,6 +75,11 @@ func ConvertNumeric(val interface{}) float64 {
 		return n
 	case float32:
 		return float64(n)
+	case bool:
+		if n {
+			return 1
+		}
+		return 0
 	default:
 		return 0
 	}
```

Comment:
nit: we could clean this up a little to not have multiple `return 0`'s by removing the default clause.

```suggestion
	case bool:
		if n {
			return 1
		}
	}
	return 0
```

Reply:
Doh, I could have just accepted your suggestion, but somehow didn't.